site/content/en/docs/Reference/agones_crd_api_reference.html

```diff
@@ -2219,6 +2219,10 @@ <h3 id="WebhookPolicy">WebhookPolicy
 
 
 
+
+
+
```

Comment:
@aLekSer I can't work out why I'm not getting the allocation group for the `GameServerAllocation` on the docs? Do you have any ideas?

Reply:
@markmandel, working on that

Reply:
Fixed the issue using `go mod edit --replace=agones.dev/agones@latest=../../../agones.dev/agones/`

Resulted webpage could be found here:
http://35.237.42.204:1313/docs/reference/agones_crd_api_reference/